tools/clang/test/CodeGenSPIRV/check-entrypoint.hlsl2spv

```diff
@@ -0,0 +1,10 @@
+// Run: %dxc -T ps_6_0 -E main
+void main()
+{
+
+}
+
+// CHECK: OpCapability Shader
+// CHECK-NOT: OpCapability Kernel
+// CHECK-NEXT: OpMemoryModel Logical GLSL450
+// CHECK-NEXT: OpEntryPoint Fragment %main "main"
```

Comment:
`CHECK-*` directives are conventionally put near to the source code generating them in LLVM instead of at the end of the file.